machines-endpoints/pkg/machines-endpoints/main.go

```diff
@@ -411,11 +415,81 @@ func main() {
 		}
 	}
 
-	if *flgBMCConfigMap {
+	if *flgBMCReverseProxyConfigMap {
 		// create bmc-proxy configmap on all servers
 		err = client.updateBMCProxyConfigMap(ctx, machines)
 		if err != nil {
 			log.ErrorExit(err)
 		}
 	}
+
+	if *flgBMCLogCollectorConfigMap {
+		// create bmc-log-collector configmap of all servers that include boot servers
+		addersses, err := client.bmcListJson(machines)
+		if err != nil {
+			log.ErrorExit(err)
+		}
+		err = client.writeConfigMap(ctx, addersses)
+		if err != nil {
+			log.ErrorExit(err)
+		}
+	}
+}
+
+func (c client) bmcListJson(machines []Machine) (map[string]string, error) {
```

Comment:
This method can be changed to a function, because it does not use client.

Reply:
Independent of the client.